code/modules/mob/living/carbon/carbon.dm

```diff
@@ -787,7 +787,7 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/unary/vent_pump,
 			buckled.user_unbuckle_mob(src,src)
 			if(istype(I, /obj/item/restraints/handcuffs/cable))
 				playsound(loc, 'sound/effects/snap.ogg', 50, 1, -1)
-				visible_message("<span class='danger'>As [src] manages to unbuckle, It destroys [I] as well!</span>")
+				visible_message("<span class='danger'>As [src] manages to unbuckle It destroys [I] as well!</span>")
```

Comment:
manages to unbuckle It destroys... "It". no sé mucho de ingles pero si vas a poner una I en mayusculas deberias poner un punto antes.

Reply:
HHAHAHA JUSTO LO ESCRIBISTES PUSE EL OTRO COMMIT

Reply:
Era mejor mí texto :pensive: